contracts/upgradeable_contracts/BaseBridgeValidators.sol

```diff
@@ -30,6 +30,37 @@ contract BaseBridgeValidators is EternalStorage, Ownable {
         return (2, 1, 0);
     }
 
+    function _addValidator(address _validator) internal onlyOwner {
```

Comment:
we don't need the modifier `onlyOwner` here since it is already checked by the caller.

Reply:
Thanks. Done on https://github.com/poanetwork/poa-bridge-contracts/pull/130/commits/a3b947a50721accd4cf7844992ca1b444a6f08c8